geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/delta/AddsDeltaInfo.java

```diff
@@ -42,10 +39,9 @@ public void add(byte[] delta) {
 
   public void serializeTo(DataOutput out) throws IOException {
     DataSerializer.writeEnum(ADDS, out);
-    DataSerializer.writeArrayList(deltas, out);
-  }
-
-  public List<byte[]> getAdds() {
-    return deltas;
+    InternalDataSerializer.writeArrayLength(deltas.size(), out);
+    for (byte[] bytes : deltas) {
+      DataSerializer.writeByteArray(bytes, out);
+    }
```

Comment:
With this change, the type of `deltas` can be changed to the interface `List<Byte[]` rather than the concrete `ArrayList<byte[]>`

Reply:
I will make this change but it will be a List of byte array not Byte.

Reply:
Sorry, yes, that was a typo.